shell/platform/windows/win32_window.cc

```diff
@@ -4,9 +4,16 @@
 
 #include "flutter/shell/platform/windows/win32_window.h"
 
+#include "dpi_utils.h"
+
 namespace flutter {
 
-Win32Window::Win32Window() {}
+Win32Window::Win32Window() {
+  // Calling GetDpiForHWND with no HNWD returns the DPI from the nearest
```

Comment:
primary

You need to get a chalkboard and write "You can't have a nearest monitor to a window that doesn't exist." 50 times :)

Reply:
😢